controllers/common.go

```diff
@@ -221,14 +221,15 @@ func (r *Reconciler) AcquireLock(ctx context.Context, instance client.Object, h
 						Namespace: instance.GetNamespace(),
 					},
 				}
-				configmap.EnsureConfigMaps(ctx, h, instance, cms, nil)
-				return true
+
+				err = configmap.EnsureConfigMaps(ctx, h, instance, cms, nil)
+				return err == nil
 			} else {
```

Comment:
could remove the else since the prior if has a return

Reply:
True, thank you!:)